website/docs/cloud-docs/api-docs/oauth-clients.mdx

```diff
@@ -456,13 +456,13 @@ curl \
   https://app.terraform.io/api/v2/oauth-clients/oc-XKFwG6ggfA9n7t1K
 ```
 
-## Attach an OAuth Client to projects
+## Attach to a project
```

Comment:
i do feel like associate is the more appropriate verb here, but can revisit later. maybe something to keep in mind as we build out adam's controlled vocab list